frontend/mock-backend/data/v1/runtime/mock-retry-runtime.ts

```diff
@@ -199,7 +199,7 @@ export default {
                   'artifacts/retry-random-failures-p9snt/retry-random-failures-p9snt-1189960944/main.log',
                 secretKeySecret: {
                   key: 'secretkey',
-                  name: 'mlpipeline-minio-artifact',
+                  name: 'mlpipeline-object-storage-artifact',
```

Comment:
For backwards compatibility the name of the secret and key must stay the same for now.